src/lib/tvgShape.cpp

```diff
@@ -349,7 +349,16 @@ uint32_t Shape::strokeDash(const float** dashPattern) const noexcept
 {
     if (!pImpl->stroke) return 0;
 
-    if (dashPattern) *dashPattern = pImpl->stroke->dashPattern;
+    if (dashPattern) {
```

Comment:
No, I mean ignore input in the setting side....

Reply:
I mean there is no point that dash has 0 values in the list (insane)
Thus, we can return fail if there is zero values in the list when user call shape->strokeDash().
Then there is no need to handle up above sides.